jetty-util/src/main/java/org/eclipse/jetty/util/thread/strategy/AdaptiveExecutionStrategy.java

```diff
@@ -345,16 +348,17 @@ private SubStrategy selectSubStrategy(Runnable task, boolean nonBlocking)
     }
 
     /**
-     * Consume a task with a sub-strategy.
+     * Consumes a task with a sub-strategy.
+     *
      * @param task The task to consume.
-     * @param subStrategy The execution sub-strategy mode to use to consume it.
+     * @param subStrategy The execution sub-strategy to use to consume the task.
      * @return True if the sub-strategy requires the caller to continue to produce tasks.
      */
     private boolean consumeTask(Runnable task, SubStrategy subStrategy)
     {
         // Consume and/or execute task according to the selected mode.
         if (LOG.isDebugEnabled())
-            LOG.debug("{} ss={} t={}/{} {}", this, subStrategy, task, Invocable.getInvocationType(task));
+            LOG.debug("ss={} t={}/{} {}", subStrategy, task, Invocable.getInvocationType(task), this);
```

Comment:
@sbordet this is now inconsistent with the other debugs in the class.... but I guess that is not so important.